pkg/api/pluginproxy/pluginproxy.go

```diff
@@ -22,10 +21,10 @@ type templateData struct {
 
 // NewApiPluginProxy create a plugin proxy
 func NewApiPluginProxy(ctx *models.ReqContext, proxyPath string, route *plugins.Route,
-	appID string, cfg *setting.Cfg, store pluginsettings.Service, secretsService secrets.Service) *httputil.ReverseProxy {
+	appID string, cfg *setting.Cfg, pluginSettingsService plugins.SettingsService, secretsService secrets.Service) *httputil.ReverseProxy {
```

Comment:
For now would probably suggest to put a `Service` interface in the `pluginsettings` package and bind the Service implementation with wire and use that as dependency in services. At least until we've decided how plugin settings and plugin management fits together.